### DISCUSSION TITLE
Anzeigen von Objekten vom Typ "file"

### DISCUSSION THREAD

**Discussion by MarcusLoy** [Zefau/ioBroker.jarvis#2342] [General]
Hallo,

gibt es eine Möglichkeit Objekte vom Typ "file" anzuzeigen?
Hier ein Auszug aus den Objekten des SONOS-Adapters:

`"_id": "sonos.0.root.1234567890.cover_png",
  "common": {
    "name": "Cover URL",
    "type": "file",
    "read": true,
    "write": true,
    "role": "media.current.cover",
    "desc": "Cover image of current played song as binary"
  }`

Früher war dort die URL des Covers hinterlegt, da war es kein Problem. Jetzt liegt leider das Bild in binärer Form drin.
Hat jemand eine Idee?

Viele Grüße
Marcus

**Comment by mcuiobroker** [comment]
Stellst du mal ein Beispiel rein. Was steht in dem DP. Danke.
Hat sich erledigt, ist normale png.

**Comment by MarcusLoy** [comment]
Gibts da eine Möglichkeit? Eher nicht, oder?
